setup.py

```diff
@@ -16,8 +16,8 @@
 
 
 REQUIREMENTS = [
-    'kinto >= 2.9',
-    'pyfxa >= 0.0.9',
+    'kinto >=4,<5',
```

Comment:
I believe so... I hadn't realized before, but I think that if it breaks plugins, we should bump a major